index.js

```diff
@@ -1,3 +0,0 @@
-exports.printMsg = function() {
```

Comment:
```suggestion
module.exports = {};
```

Otherwise, `require('buhari')` will throw an exception. Wait, is that the intention?

Reply:
I mean, isn't he an exception?

Is there anything worth exporting from the guy?

Reply:
An object is still something hence removing the empty line lol.

Let me add that line back.

Reply:
:laughing:

Reply:
I just create a pull request. This should be a nightmare package for slowing down systems or application. The next malware can start like this. I think people should just submit disastrous logics that can slow down fastest script

Reply:
@olyjosh The script does something then lol